lib/text_sentinel.rb

```diff
@@ -45,7 +45,7 @@ def valid?
     seems_meaningful? &&
     seems_pronounceable? &&
     seems_unpretentious? &&
-    seems_quiet?
+    (SiteSetting.allow_uppercase_posts || seems_quiet?)
```

Comment:
To make it consistent with the the other `seems_` methods, could you move the `SiteSetting` check inside the `seems_quiet?` -- it'll just read better